Resources/public/js/views/fields/ez-relation-editview.js

```diff
@@ -0,0 +1,195 @@
+                'tap': '_runUniversalDiscovery',
+            },
+        },
+
+        initializer: function () {
+            var that = this;
+
+            this._fireMethod = this._fireLoadFieldRelatedContent;
+            this._watchAttribute = 'destinationContent';
+            if( this.get('field').fieldValue.destinationContentId ){
+                this._set('destinationContentId', this.get('field').fieldValue.destinationContentId);
+            }
+            this.after("destinationContentChange", function () {
+                if ( that.get('destinationContent') ) {
+                    that._set('destinationContentId', that.get('destinationContent').get('contentId'));
+                } else {
+                    that._set('destinationContentId', null);
+                }
+            });
+        },
+
+        /**
+         * Fire the `loadFieldRelatedContent` event
+         *
+         * @method _fireLoadFieldRelatedContent
+         * @protected
+         */
+        _fireLoadFieldRelatedContent: function () {
+            if ( !this._isFieldEmpty() ) {
+                this.fire('loadFieldRelatedContent', {
+                    fieldDefinitionIdentifier: this.get('fieldDefinition').identifier
+                });
+            }
+        },
+
+        /**
+         * Checks whether the field is empty
+         *
+         * @method _isFieldEmpty
+         * @protected
+         * @return {Boolean}
+         */
+        _isFieldEmpty: function () {
+            return !this.get('destinationContentId');
+        },
+
+        /**
+         * Returns an object containing the additional variables
+         *
+         * @method _variables
+         * @protected
+         * @return Object
+         */
+        _variables: function () {
+            var dest = this.get('destinationContent');
+
+            return {
+                destinationContent: dest ? dest.toJSON() : null,
+                loadingError: this.get('loadingError'),
+                isEmpty: this._isFieldEmpty(),
+                isRequired: this.get('fieldDefinition').isRequired,
+            };
+        },
+
+        /**
+         * Tap event handler for the remove button. It resets the relation.
+         *
+         * @method _removeRelation
+         * @protected
+         * @param {EventFacade} e
+         */
+        _removeRelation: function (e) {
+            e.preventDefault();
+
+            this.set('destinationContent', null);
+            this.validate();
+            this.get('container').one('.ez-relationeditview-remove').set('disabled', true);
+        },
+
+        validate: function () {
+            if ( this.get('fieldDefinition').isRequired && this._isFieldEmpty() ){
+                this.set('errorStatus', 'This field is required');
+            } else {
+                this.set('errorStatus', false);
+            }
+        },
+
+        /**
+         * Fire the contentDiscover event with the universalDiscovery.
```

Comment:
*with the universalDiscovery* ? Better *Fire the contentDiscover event to launch the universal discovery widget*